packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPictureTile.tsx

```diff
@@ -1,40 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { memoizeFunction } from '@fluentui/react';
-import React from 'react';
+import { IStackStyles, memoizeFunction, Stack } from '@fluentui/react';
+import React, { ReactChild } from 'react';
 import { useTheme } from '../../theming/FluentThemeProvider';
-import { VideoTile, VideoTileProps, VideoTileStylesProps } from '../VideoTile';
 
 /** @internal */
 export type _TileOrientation = 'portrait' | 'landscape';
 
 /** @internal */
-export interface _PictureInPictureInPictureTileProps
-  extends Pick<
-    VideoTileProps,
-    'styles' | 'displayName' | 'renderElement' | 'isMirrored' | 'noVideoAvailableAriaLabel'
-  > {
+export interface _PictureInPictureInPictureTileProps {
+  content: ReactChild;
```

Comment:
This can just be `children` right? We expect exactly one child and we simply forward it to the `Stack` in the implementation.

Reply:
I haven't really understood the 'children' naming when it has to be a single child. There is a react.children type but we don't use it anywhere (we typically use react.reactnode for children)

Reply:
`props.children` is perfect for cases where all your component does is forward the children as children to another component (in this case a `Stack`).

https://reactjs.org/docs/glossary.html#propschildren

It does mean that your Component can not specify there should be only one child. But does that matter for this component? If multiple children are provided, the `Stack` will lay them out.

The big benefit in using `props.children` is how React handles children during reconciliation. A change in a prop for `_PictureInPictureInPictureTileProps` does not cause the childrens' `render()` method getting called. If you pass it in the child as a prop and render it inside, then it will.